ui/TxFrames.py

```diff
@@ -1729,29 +1725,29 @@ def signTx(self):
 
    def broadTx(self):
       if self.main.netMode == NETWORKMODE.Disconnected:
-         QMessageBox.warning(self, 'No Internet!', \
-            'Armory lost its connection to Bitcoin-Core, and cannot '
+         QMessageBox.warning(self, self.tr('No Internet!'), \
+            self.tr('Armory lost its connection to , and cannot '
```

Comment:
Bitcoin-Core removed a whole in the few following sentences. No rephrasing though.